source/_components/lcn.markdown

```diff
@@ -271,8 +296,23 @@ The [MOTOR_PORT](/components/lcn#ports) values specify which hardware relay conf
 | LED_STATE | `on`. `off`, `blink`, `flicker` |
 | LOGICOP_STATE | `not`. `or`, `and` |
 
+### {% linkable_title Keys %}:
+
+Whenever a key has to be provided, it is defined by a joint string consisting of the table identifier (`a`, `b`, `c`, `d`) and the corresponding key number.
+Examples: `a1`, `a5`, `d8`.
```

Comment:
These keys have uppercase letters in the implementation, is that an intended mismatch?

Reply:
Usually the values in the configuration.yaml are lowercase. Same here.
But the underlying library (‘pypck‘) uses uppercase values for enumerator keys. The conversion is done in the schema-validator.
So yes, the mismatch is intended to keep up with the ‘lowercase configuration values standard‘.

Reply:
Awesome, I missed the bit about `vol.Upper` in the schema.